modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift-sdn/configuring-egress-firewall.adoc
+
+[id="nw-egressnetworkpolicy-about_{context}"]
+= How an egress firewall works in a project
+
+As an {product-title} cluster administrator, you can use an _egress firewall_ to
+limit the external hosts that some or all Pods can access from within the
+cluster. Egress firewall policy supports the following scenarios:
```

Comment:
Is it the "egress firewall" or "egress network?" I feel like you need an article before egress.